c10/core/impl/LocalDispatchKeySet.h

```diff
@@ -24,67 +25,45 @@
 namespace c10 {
 namespace impl {
 
-// POD version of LocalDispatchKeySet.  Declared here just so that
-// we can put it in the guards.
-// This struct encapsulates special handling for TLS initialization
-// in set_included()/included() API so that they reflect the truth.
-// If you want to create PODLocalDispatchKeySet with non-zero state,
-// use set_included() instead of default constructor.
-struct C10_API PODLocalDispatchKeySet {
-  uint64_t included_;
-  uint64_t excluded_;
+// Helper class to handle conversion between the TLS zero initialized
+// convention and the "normal" in memory layout.
```

Comment:
It's not just this right, idiomatic usage seems to involve the default constructor, which does a TLS fetch. (To be honest, I'd prefer it if it were more obvious when a TLS fetch was happening, rather than doing so implicitly)

Reply:
SGTM